### PR TITLE
Rename cardano-wallet-shelley to cardano-wallet

### PR DESCRIPTION
**Pull request by Anviking**
# Issue Number

#1919 

# Overview

- [x] Rename `cardano-wallet-shelley` to `cardano-wallet`
- [x] Check that everything still works (Think it might do)
- [ ] Update wiki 

# Comments

Package structure after renaming as a reminder:
wreq

[Hydra jobset](https://hydra.iohk.io/jobset/Cardano/cardano-wallet-pr-2013#tabs-jobs)

### CODE REVIEW

**Comment by Anviking** [comment]
bors try

File: .travis.yml [hunk 1]
```diff
@@ -54,26 +54,26 @@ jobs:
     ############################
 
     # Fetch the linux archive
-    - travis_retry curl -L https://hydra.iohk.io/job/Cardano/cardano-wallet/cardano-wallet-shelley-linux64/latest/download/1 | tar xz
-    - find . -maxdepth 1 -type d -name "cardano-wallet-shelley-*" -exec mv \{} cardano-wallet-shelley-linux64 \;
-    - mv cardano-wallet-shelley-linux64/cardano-wallet-shelley cardano-wallet-shelley-linux64/cardano-wallet
-    - tar czf cardano-wallet-$TRAVIS_TAG-linux64.tar.gz cardano-wallet-shelley-linux64
-    - rm -rf cardano-wallet-shelley-linux64
+    - travis_retry curl -L https://hydra.iohk.io/job/Cardano/cardano-wallet/cardano-wallet-linux64/latest/download/1 | tar xz
+    - find . -maxdepth 1 -type d -name "cardano-wallet-*" -exec mv \{} cardano-wallet-linux64 \;
```
Comment (Anviking) [inline .travis.yml:58]: This `cardano-wallet-*` will probably stop working though assuming the binaries from hydra will change

**Review by rvl — APPROVED** [review]
If it builds then good. The nightly tests might also need testing/tweaking.

Since you're already making such a change, what about cleaning up another naming/paths issue at the same time?

All of our code is in the `lib` subdirectory. There is no need for this. The packages could be moved to the top level.

**Comment by Anviking** [comment]
> The packages could be moved to the top level.

Can be [discussed](https://input-output-rnd.slack.com/archives/GBT05825V/p1597133187340900?thread_ts=1597051716.330400&cid=GBT05825V) and done in another PR. I made a partially compiling commit, but dropped it. In the meantime:
bors r+

File: .travis.yml (hunk 1 above)
Comment (Anviking) [reply .travis.yml:58]: Will deal with it when merged and the latest-hydra-link is updated.

**Pull request merged**